docs/csharp/event-pattern.md

```diff
@@ -270,7 +270,7 @@ need extra code in those handlers in this project, but this shows how
 you would create them.
 
 ```cs
-internal event EventHandler<SearchDirectoryArgs> OnChangeDirectory
+internal event EventHandler<SearchDirectoryArgs> ChangeDirectory
```

Comment:
Wouldn't `DirectoryChanged` be a better name? It would also be more consistent with `FoundFile`.

Also, why is `changeDirectory` just below a `private event`? Wouldn't a field be enough?